content/docs/reference-events.md

```diff
@@ -52,70 +52,70 @@ function onClick(event) {
     console.log(eventType); // => "click"
   }, 0);
 
-  // Won't work. this.state.clickEvent will only contain null values.
+  // Ne fonctionnera pas. this.state.clickEvent ne contiendra que des valeurs à null.
   this.setState({clickEvent: event});
 
-  // You can still export event properties.
+  // Vous pouvez toutefois exporter les propriétés de l'événement.
   this.setState({eventType: event.type});
 }
 ```
 
-> Note:
+> Note :
 >
-> If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.
-
-## Supported Events {#supported-events}
-
-React normalizes events so that they have consistent properties across different browsers.
-
-The event handlers below are triggered by an event in the bubbling phase. To register an event handler for the capture phase, append `Capture` to the event name; for example, instead of using `onClick`, you would use `onClickCapture` to handle the click event in the capture phase.
-
-- [Clipboard Events](#clipboard-events)
-- [Composition Events](#composition-events)
-- [Keyboard Events](#keyboard-events)
-- [Focus Events](#focus-events)
-- [Form Events](#form-events)
-- [Mouse Events](#mouse-events)
-- [Pointer Events](#pointer-events)
-- [Selection Events](#selection-events)
-- [Touch Events](#touch-events)
-- [UI Events](#ui-events)
-- [Wheel Events](#wheel-events)
-- [Media Events](#media-events)
-- [Image Events](#image-events)
-- [Animation Events](#animation-events)
-- [Transition Events](#transition-events)
-- [Other Events](#other-events)
+> Si vous souhaitez accéder aux propriétés de l'événement de façon asynchrone, vous devez appeler la fonction `event.persist()` de l'événement, ce qui aura pour effet de supprimer l'événement synthétique du partage, et permettra l'utilisation de la référence à l'événement au sein de votre code.
+
+## Événements supportés {#supported-events}
+
+React normalise les événements de façon à ce qu'ils conservent des propriétés cohérentes entre les différents navigateurs.
```

Comment:
```suggestion
React normalise les événements pour qu’ils aient les mêmes propriétés dans tous les navigateurs.
```

- « de fa!on à ce » s'allège facilement en « pour »
- Ici, on utilise “consistent” dans le sens moins courant, qui ne focalise pas tant sur la cohérence (qui pourrait ici être perçu comme une cohérence interne de l'événement) que sur l'homogénéité (caractère identique à travers différents contextes, ici les navigateurs). On préfère bien retranscrire ce point, ce qui allège en prime la traduction.